From ce586ba991967b1a663a9082312a45725da6fe5b Mon Sep 17 00:00:00 2001
From: Kalev Lember <kalevlember@gmail.com>
Date: Wed, 28 Nov 2012 23:25:52 +0100
Subject: [PATCH] Unconditionally use g_content_type_get_symbolic_icon()

... and g_content_type_get_generic_icon_name(). The new functions are
implemented for Win32 since commit dace477c, so we no longer need to
guard them with G_OS_UNIX.
---
 gio/gio.symbols      | 2 --
 gio/glocalfileinfo.c | 2 --
 2 files changed, 4 deletions(-)

diff --git a/gio/gio.symbols b/gio/gio.symbols
index 860e3cb..6788fc1 100644
--- a/gio/gio.symbols
+++ b/gio/gio.symbols
@@ -165,10 +165,8 @@ g_content_type_is_unknown
 g_content_type_get_description
 g_content_type_get_mime_type
 g_content_type_get_icon
-#ifdef G_OS_UNIX
 g_content_type_get_symbolic_icon
 g_content_type_get_generic_icon_name
-#endif
 g_content_type_can_be_executable
 g_content_type_from_mime_type
 g_content_type_guess
diff --git a/gio/glocalfileinfo.c b/gio/glocalfileinfo.c
index 9c97495..9cfa041 100644
--- a/gio/glocalfileinfo.c
+++ b/gio/glocalfileinfo.c
@@ -1535,11 +1535,9 @@ get_icon (const char *path,
     }
   else
     {
-#ifdef G_OS_UNIX
       if (use_symbolic)
         icon = g_content_type_get_symbolic_icon (content_type);
       else
-#endif
         icon = g_content_type_get_icon (content_type);
 
       if (G_IS_THEMED_ICON (icon) && is_folder)
-- 
1.8.0

